cf/aggregate.py

```diff
@@ -3505,10 +3514,15 @@ def _create_hash_and_first_values(
     # identity.
     canonical_direction = {}
 
+    aggregating_axes = list(axes)
```

Comment:
If we've already specified this input should be a `list`, we don' t really need to do this conversion, though I appreciate this might be for safety in case of another sequence input. I'd personally remove it, but it is up to you:

```suggestion
    aggregating_axes = axes
```

Reply:
Good point. Contrary to what I said I thought was the case, it is much cleaner to just replace the `axes` input parameter with `aggregating_axes`. Done. 4f023d6063fedecfb824258a663df7cfac05a588